### DISCUSSION TITLE
Not able to introspect my database

### DISCUSSION THREAD

**Discussion by ccordon01** [prisma/prisma#17484] [Q&A]
I'm been trying to introspect my database with prisma but I'm getting this error:
```
> npx prisma db pull
Prisma schema loaded from schema.prisma
Environment variables loaded from .env
Datasource "db": MySQL database "test_db" at "localhost:3306"

✖ Introspecting based on datasource defined in schema.prisma

Error: spawn EPERM
```
Currently I'm using this versions in my environment
Node.js: v16.19.0 (I also try with v18.13.0, both LTS version)
Prisma: 4.9.0
This error begun when I updated prisma to v4.9.0 I was using v4.8.1

**Comment by nurul3101** [comment]
Hey @ccordon01 👋 
So is this error occurring only on Prisma `v4.9.0` and wasn't happening on `v4.8.1`?

Can you try setting the [DEBUG environment variable](https://www.prisma.io/docs/concepts/components/prisma-client/debugging#setting-the-debug-environment-variable) and check if you get more information about the error?

**Reply by ccordon01** [reply]
Thanks for your reply

Yes, this is only happening with the latest version of prisma (`v4.9.0`)

I'm not getting to much details using DEBUG env, but the console throw this:

```
⠋ Introspecting based on datasource defined in schema.prisma  prisma:migrateEngine:rpc  
starting migration engine with binary: \node_modules\@prisma\engines\migration-engine-windows.exe +0ms
✖ Introspecting based on datasource defined in schema.prisma

Error: Error: spawn EPERM
    at ChildProcess.spawn (node:internal/child_process:420:11)
    at spawn (node:child_process:733:9)
    at C:\Users\carlos.cordon\Documents\dev\MAYFER\MAYRRHH\main-api\node_modules\prisma\build\index.js:96831:54
```

**Reply by nurul3101** [reply]
Thanks for getting back. Can you please create a [Bug Report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that our engineering team can have a look?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
